src/control/server/security_rpc_test.go

```diff
@@ -71,13 +72,12 @@ func TestSrvSecurityModule_HandleCall_BadMethod(t *testing.T) {
 	defer common.ShowBufferOnFailure(t, buf)
 
 	mod := NewSecurityModule(log, insecureTransportConfig())
-	resp, err := mod.HandleCall(nil, -1, nil)
-
-	if resp != nil {
-		t.Errorf("Expected no response, got %+v", resp)
+	method, err := mod.ID().GetMethod(-1)
+	if method != nil {
+		t.Errorf("Expected no method to be returned, got %+v", method)
```

Comment:
Just want to point out that this changes the test... It's no longer testing what happens when an invalid method is passed to that module's `HandleCall()` implementation.

Reply:
So yes, because that situation now isn't possible. instead it's now testing that the expected error is returned when the method lookup fails and that the returned method is nil.

Reply:
Sure, but the name of the test is `TestSrvSecurityModule_HandleCall_BadMethod`. Not objecting to the change, just that the test purports to be for testing one thing but is now testing something else.

Reply:
will update, thanks

Reply:
fixing in https://github.com/daos-stack/daos/pull/2730